src/main/kotlin/com/redhat/devtools/intellij/kubernetes/settings/SettingsConfigurable.kt

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
```

Comment:
Why do we have a settings ? The state is empty and it does not align with others plugins (ex OpenShift) ?

Reply:
I dont get your point. Ex. tekton also has a `Configurable`. By the interface it looks as if it is the one responsible for creating the JPanel:
```
public @Nullable JComponent createComponent() {
```
What am I missing?

Reply:
Tekton has a configurable because it has specific options that are used by various actions (wizards,...) from the plugin. But the OpenShift plugin does not have a configurable and it telemetry enabled

Reply:
@jeffmaury so you suggest to not have settings at all? I thought that you were complaining about the presence of the settings configurable. Sure can do that, there's no need for settings.

Reply:
Yes if there is no settings for the plugin then no configurable